app/src/renderer/vuex/modules/delegation.js

```diff
@@ -178,6 +178,7 @@ export default ({ node }) => {
         value: state.committedDelegates[validator_address] + Number(amount)
       })
 
+      await dispatch(`getAllTxs`)
```

Comment:
should be deferred until the next block porbably right?

Reply:
you suggest setting a timeout ? or removing the await ?

Reply:
nvm, I think our send waits for the tx to be included in a block
will be harder wenn send is async.